<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tusvir</class>
 <widget class="QMainWindow" name="Tusvir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tusvir</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>241</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="FileMenu">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Open File</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>731</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>30</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="originalPicture">
     <item>
      <widget class="QGraphicsView" name="OriginalPictureDisp"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>260</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="histogram">
     <item>
      <widget class="QGraphicsView" name="HistogramDisp"/>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollBar" name="histogramScroller">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>490</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>199</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>550</y>
      <width>211</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1220</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
